sled-agent/types/src/early_networking.rs

```diff
@@ -323,8 +323,8 @@ pub mod back_compat {
         pub uplink_cidr: Ipv4Net,
         /// VLAN id to use for uplink
         pub uplink_vid: Option<u16>,
-        /// Local preference
-        pub local_pref: Option<u32>,
+        /// RIB Priority
+        pub rib_priority: Option<u8>,
```

Comment:
I see that this struct was deprecated; was updating it the right thing to do?

Reply:
Good catch. @internet-diglett can you take a look at this PR from the perspective of bootstore compatibility?

Reply:
In theory it should be harmless at this point, but ahl is correct that it probably shouldn't have been updated.